packages/telemetry/src/telemetry.ts

```diff
@@ -4,12 +4,15 @@ import path from 'path'
 
 import { getPaths } from '@redwoodjs/internal/dist/paths'
 
-const APP_ROOT = getPaths().base
```

Comment:
Calling this at the top level means it executes as soon as it's imported. That's too soon for the CLI. We could also just import `@redwoodjs/telemetry` asynchronously. I didn't have a reason to prefer one over the other.